backend/src/ml_space_lambda/report/lambda_functions.py

```diff
@@ -28,7 +28,7 @@
 from ml_space_lambda.data_access_objects.resource_scheduler import ResourceSchedulerDAO, ResourceSchedulerModel
 from ml_space_lambda.data_access_objects.user import UserDAO, UserModel
 from ml_space_lambda.enums import Permission, ResourceType
-from ml_space_lambda.utils.common_functions import api_wrapper, get_tags_for_resource, list_clusters_for_project, retry_config
+from ml_space_lambda.utils.common_functions import api_wrapper, retry_config
```

Comment:
Not totally related to this PR but can you remove the EMR client (line 42)? It shouldn't be used any more. The sagemaker client (line 39) can also be removed.

Reply:
Good call, neither is in use.